scripts/onboarding/aks/onboarding-using-legacy-auth/existingClusterOnboarding.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "aksResourceId": {
+            "type": "string",
+            "metadata": {
+                "description": "AKS Cluster Resource ID"
+            }
+        },
+        "aksResourceLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "Location of the AKS resource e.g. \"East US\""
+            }
+        },
+        "workspaceResourceId": {
+            "type": "string",
+            "metadata": {
+                "description": "Azure Monitor Log Analytics Resource ID"
+            }
+        }
+    },
+    "resources": [
+        {
+            "name": "[split(parameters('aksResourceId'),'/')[8]]",
+            "type": "Microsoft.ContainerService/managedClusters",
+            "location": "[parameters('aksResourceLocation')]",
+            "apiVersion": "2018-03-31",
+            "properties": {
+                "mode": "Incremental",
+                "id": "[parameters('aksResourceId')]",
+                "addonProfiles": {
+                    "omsagent": {
+                        "enabled": false,
```

Comment:
these templates should be used for backdoor testing should be moved under "tests" directory.